docs/maintain/maintain-guides-how-to-chill.md

```diff
@@ -25,63 +23,35 @@ staking, take a look at the [accounts][] section in the general staking guide.
 :::
 
 ![staking](../assets/NPoS/staking-keys_stash_controller.png)
+## Consideration for Staking Election
+
+A bond that is actively participating in staking but chilled would continue to participate in staking for the rest of the current era.  If the bond was chilled in sessions 1 through 4 and continues to be chilled for the rest of the era it would NOT be selected for election in the next era.  If a bond was chilled for the entire of session 5 it would not be considered in the next election.  If the bond was chilled in session 6 its participation in the next era's election would be dependent on its state in session 5.
 
 ## Chilling as a Nominator
 
-When you chill after being a nominator, your nominations will be reset. This means that when you
-decide to start nominating again you will need to select validators to nominate once again. These
-can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations
-will not persist across chills.
+When you chill after being a nominator, your nominations will be reset. This means that when you decide to start nominating again you will need to select validators to nominate once again. These can be the same validators if you prefer, or, a completely new set. Just be aware - your nominations will not persist across chills.
 
-Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will
-not need to go through the whole process of bonding again, rather, you will issue a new nominate
-call that specifies the new targets to nominate.
+Your nominator will remain bonded when it is chilled. When you are ready to nominate again, you will not need to go through the whole process of bonding again, rather, you will issue a new nominate call that specifies the new validators to nominate.
 
 ## Chilling as a Validator
 
-When you voluntarily chill after being a validator, your nominators will not automatically go away.
-As long as your nominators make no action, you will still have the nominations when you choose to
-become an active validator once again. However, if your nominators decide to nominate other
-validators then these nominations will take priority when the validator comes back. It may also be
-the case that your nominators change their entire nomination targets (all 16 of the allowed
-nominations). In this case, your nominators would need to explicitly specify your validator as a
-target when your validator comes back.
+When you voluntarily chill after being a validator, your nominators will remain.  As long as your nominators make no action, you will still have the nominations when you choose to become an active validator once again.  You bond however would not be listed as a nominable validator thus any nominators issuing new or revisions to existing nominations would not be able to select your bond.
```

Comment:
By bond, do you mean the validator node?

```suggestion
When you voluntarily chill after being a validator, your nominators will remain.  If your nominators take no action, you will still have their nominations when you choose to become an active validator again.  Your node, however, would not be listed as a nominable validator; thus, any nominators issuing new or revisions to existing nominations would be unable to select your node.
```

Reply:
It is probably just how I see things.  Nominators nominate the validator's bond.  The bond references the physical node via session keys.  Does that sound right or do you think it is too confusing?

Reply:
As the target audience for this doc is validators, this should be okay!